trade.lic

```diff
@@ -1298,7 +1298,8 @@ class Trade
     return unless time_to_room(Room.current.id, @trading_towns[near_town]['deposit']['id']) < 20.0
 
     walk_to(@town_data[near_town]['deposit']['id'])
-    minimize_coins(withdraw_amt).each { |amount| get_money_from_specific_bank?(amount, near_town) }
+    #minimize_coins(withdraw_amt).each { |amount| get_money_from_specific_bank?(amount, near_town) }
+    minimize_coins(withdraw_amt).each { |amount| withdraw_exact_amount?(amount, @settings, "#{near_town}") }
```

Comment:
Do you need the string interpolation here?

Reply:
Yes. I tried without it and it did not work.

Reply:
In hometown depositing and withdrawing (I did not need any items from Crossing):
```2021-01-04 18:26:43 -0500:Wealth:
2021-01-04 18:26:44 -0500:  3 silver Kronars (300 copper Kronars).
2021-01-04 18:26:44 -0500:  No Lirums.
2021-01-04 18:26:44 -0500:  No Dokoras.
2021-01-04 18:26:44 -0500:[restock]>deposit all
2021-01-04 18:26:44 -0500:You sense that your familiar is traveling.
2021-01-04 18:26:44 -0500:The clerk slides a small metal box across the counter into which you drop all your Kronars.  She counts them carefully and records the deposit in her ledger.
2021-01-04 18:26:44 -0500:[drinfomon]>balance
2021-01-04 18:26:44 -0500:[restock]>withdraw 3 silver
2021-01-04 18:26:45 -0500:The clerk pages through her ledger.  "Waddle, Dartellum," she says to herself.  "Here we are.  Your current balance is xxx platinum,xx gold, x silver, x bronze, and x copper Kronars."
2021-01-04 18:26:45 -0500:The clerk counts out 3 silver Kronars and hands them over, making a notation in her ledger.
2021-01-04 18:26:45 -0500:[restock]>check balance
```

In town defined as hometown by restock setting hometown:
```021-01-03 17:04:17 -0500:[First Bank of Ilithi, Teller's Windows]
2021-01-03 17:04:17 -0500:Polished walnut windows and paneling separate the customers from the tellers and vault against the back of the wall.  Several young and pretty Elothean men and women stand behind bars of an anlora-avtoma and steel alloy to process transactions.  A gleaming brass sign is embedded into the west wall, stating the bank's policies.  You also see a small calendar.
2021-01-03 17:04:17 -0500:Also here: Friar Shimahashi.
2021-01-03 17:04:17 -0500:Obvious exits: south.
2021-01-03 17:04:17 -0500:Room Number: 19276
2021-01-03 17:04:17 -0500:[go2: travel time: 0:00:59]
2021-01-03 17:04:17 -0500:--- Lich: go2 has exited.
2021-01-03 17:04:18 -0500:You feel fully rested.
2021-01-03 17:04:18 -0500:[restock]>withdraw 22099 copper
2021-01-03 17:04:18 -0500:The clerk counts out 22,099 copper Dokoras and hands them over, making a notation in her ledger.
2021-01-03 17:04:43 -0500:[First Bank of Ilithi, Teller's Windows]
2021-01-03 17:04:43 -0500:Polished walnut windows and paneling separate the customers from the tellers and vault against the back of the wall.  Several young and pretty Elothean men and women stand behind bars of an anlora-avtoma and steel alloy to process transactions.  A gleaming brass sign is embedded into the west wall, stating the bank's policies.  You also see a small calendar.
2021-01-03 17:04:43 -0500:Also here: Friar Shimahashi.
2021-01-03 17:04:43 -0500:Obvious exits: south.
2021-01-03 17:04:43 -0500:Room Number: 19276
2021-01-03 17:04:44 -0500:[go2: travel time: 0:00:01]
2021-01-03 17:04:44 -0500:--- Lich: go2 has exited.
2021-01-03 17:04:44 -0500:[restock]>wealth
2021-01-03 17:04:44 -0500:Wealth:
2021-01-03 17:04:44 -0500:  3 silver Kronars (300 copper Kronars).
2021-01-03 17:04:44 -0500:  No Lirums.
2021-01-03 17:04:44 -0500:  No Dokoras.
2021-01-03 17:04:44 -0500:[restock]>deposit all
2021-01-03 17:04:45 -0500:You don't have any Dokoras to deposit.
```
I withdrew the proper amount for this run. :smile: